app/src/main/java/com/kickstarter/ui/activities/CreatorBioActivity.kt

```diff
@@ -4,35 +4,39 @@ import android.content.Intent
 import android.os.Bundle
```

Comment:
Creator Bio not loading, but on latest master it is loading

https://user-images.githubusercontent.com/4083656/128579689-67118677-7d29-4f4b-b51f-8ed4912ecc15.mp4

Reply:
Fixed as the issue was in web view